### ISSUE TITLE
Problem in 'make' procedure for Ubuntu 

### ISSUE THREAD

**Issue by GeorgeGkas** [wiire/pixiewps#13]
I use Ubuntu 15.04 and I just download the source. When I try to 'make' the program I get the following errors:

```
cc -std=c99 -O3 -Wno-deprecated-declarations -o pixiewps pixiewps.c random_r.c -lcrypto -I../include
In file included from pixiewps.c:57:0:
pixiewps.h:66:25: fatal error: openssl/sha.h: No such file or directory
 #include <openssl/sha.h>
                         ^
compilation terminated.
In file included from random_r.c:25:0:
../include/features.h:256:0: warning: "__STDC_IEC_559__" redefined
 #define __STDC_IEC_559__  1
 ^
In file included from <command-line>:0:0:
/usr/include/stdc-predef.h:38:0: note: this is the location of the previous definition
 #  define __STDC_IEC_559__  1
 ^
In file included from random_r.c:25:0:
../include/features.h:257:0: warning: "__STDC_IEC_559_COMPLEX__" redefined
 #define __STDC_IEC_559_COMPLEX__ 1
 ^
In file included from <command-line>:0:0:
/usr/include/stdc-predef.h:46:0: note: this is the location of the previous definition
 #  define __STDC_IEC_559_COMPLEX__ 1
 ^
In file included from random_r.c:25:0:
../include/features.h:259:0: warning: "__STDC_ISO_10646__" redefined
 #define __STDC_ISO_10646__  200009L
 ^
In file included from <command-line>:0:0:
/usr/include/stdc-predef.h:54:0: note: this is the location of the previous definition
 #define __STDC_ISO_10646__  201103L
 ^
Makefile:11: recipe for target 'all' failed
make: *** [all] Error 1
```
I think the problem is in `crypto` parameter of the cc compiler but I don't know what to replace to work.

**Comment by wiire** [comment]
I think you're missing `libssl-dev`. Tried now on Kubuntu 15.04 and works despite the warnings.

Did you `sudo apt-get install libssl-dev`?

**Comment by GeorgeGkas** [comment]
`libssl-dev is already the newest version.`
I installed all the dependences. And I downloaded the zip from the repository, but even with `sudo make` I get the above errors.

**Comment by GeorgeGkas** [comment]
Ok I installed it. But I don't understand why the warnings appear in `make` procedure. It  would be nice if it all gone. I though that the `make` procedure was unsuccessful but all in all it was just warnings.

**Comment by wiire** [comment]
Glad you made it. A note that may help others; the actual error was:

`pixiewps.h:66:25: fatal error: openssl/sha.h: No such file or directory`

The others were just warnings which will not prevent from compiling. If you are unsure if the compilation was successful, check the `pixiewps/src/` directory and if you find an executable called `pixiewps` then the compilation was good.

As stated in the [README.md](https://github.com/wiire/pixiewps/blob/master/README.md), pixiewps requires `libssl-dev`. You can verify this by running:

```
sudo apt-get install apt-file
apt-file update
apt-file search 'openssl/sha.h'
```

and you should get something like: `libssl-dev: /usr/include/openssl/sha.h`, which tells you that `libssl-dev` is a requirement.

The warnings come from an header file which was included for Mac OS compatibility. I'll try find a fix for those.

**Comment by wiire** [comment]
Warnings issue fixed (c849ad1).
